src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -207,6 +212,28 @@ public TaskData getTaskByID(long taskID) {
     return new ArrayList<>();
   }
 
+  //   public ArrayList<TaskData> getParents(String parentsString) {
+  //     List<Long> parentIDs =
+  //         new ArrayList<>(Arrays.asList(parentsString.split("/")))
+  //             .stream()
+  //             .map(Long::parseLong)
+  //             .collect(Collectors.toList());
+  //     Filter filter = new FilterPredicate("taskID", FilterOperator.IN, parentIDs);
+  //     return getTasks(filter, NO_QUERY_LIMIT, NO_QUERY_SORT);
+  //   }
+
+  public ArrayList<TaskData> getParents(TaskData task) {
+    ArrayList<TaskData> parentTasks = new ArrayList<>();
+    long parentTaskID = task.getParentTaskID();
+    while (parentTaskID != 0) {
```

Comment:
Do subtasks have the projectID set?  If so, you might be able to build this relationship by loading all tasks in a project

Reply:
Yes, I might do that in the future if I have enough time.